site/source/cookbook/widgets/lazy-load-widget.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+category: widgets
+title: Lazy load a widget
+overview: Lazy load a widget on demand and exclude it from the main JavaScript bundle
+---
+
+## Objective
+
+This recipe demonstrates the following:
+
+1. How to declare a lazily loaded widget
+2. How to exclude a widget from the main JavaScript bundle
```

Comment:
Should we say which part of this enables this and what actually does it? I.e. it's defining an item in registry with a function that dynamically imports the widget in question and the cli-build cli command detects this and automatically creates a separate bundle.

Reply:
Added a note on which steps relate to these points